packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -660,6 +660,68 @@ describe('NotificationDetail Page', async () => {
     vi.useRealTimers();
   });
 
+  it('should not duplicate payments when api call to cart respond with an error', async () => {
+    vi.useFakeTimers();
+    const paymentHistory = populatePaymentsPagoPaF24(
```

Comment:
This test is a false positive. You are checking that a const, that is external from the component flux, would be updated.

Remove this

Reply:
don't understand